js/main.js

```diff
@@ -0,0 +1,24 @@
+var $photoImage = document.querySelector('.journal-photo > img');
+
+var $inputURL = document.querySelector('#photo');
+
+$inputURL.addEventListener('input', function () {
+  $photoImage.setAttribute('src', event.target.value);
+});
+
+var $formJournal = document.querySelector('form');
+
+$formJournal.addEventListener('submit', save);
+
+function save(event) {
+  event.preventDefault();
+  var formInput = {};
+  formInput.title = $formJournal.elements.title.value;
+  formInput.photo = $formJournal.elements.photo.value;
+  formInput.notes = $formJournal.elements.notes.value;
+  formInput.entryId = data.nextEntryId;
+  data.nextEntryId += 1;
+  data.entries.push(formInput);
```

Comment:
You'll want to use `unshift()` here, so that the newest entry will always be at the beginning. This will tie into when you move on to rendering the entries